src/components/Header.jsx

```diff
@@ -32,46 +34,71 @@ export class Header extends React.Component {
-)}
-              rootClose
-            >
-              <span className="user-profile ml-auto"><img className="rounded-circle" src="https://marriedbiography.com/wp-content/uploads/2018/01/Enrique-Iglesias.jpg" alt="user" /></span>
-            </OverlayTrigger>
-          ) : (
-            <Nav className="ml-auto">
-              <Nav.Item
-                onClick={this.dispatchLogin}
-                className="nav-link auth-btn login"
+          <Nav className="ml-auto">
+            <Nav.Item className="search-placement">
+              <SearchForm />
+            </Nav.Item>
+            {authenticated ? (
+              <OverlayTrigger
+                trigger="click"
+                placement="bottom"
+                overlay={(
+                  <Popover className="user-profile-container ml-auto">
+                    <Link
+                      to="/article/create"
+                      href="/article/create"
+                      className="nav-dropdown"
+                    >
+                      New Article
+                    </Link>
+                    <br />
+                    <br />
+                    <Link
+                      to="/articles/drafts"
+                      href="/article/create"
+                      className="nav-dropdown"
+                    >
+                      Drafts
+                    </Link>
+                    <hr />
+                    <Link
+                      to="/profile"
+                      href="/profile"
+                      className="nav-dropdown"
+                    >
+                      Profile
+                    </Link>
+                    <hr />
+                    <span
+                      className="logout nav-dropdown"
+                      onClick={this.dispatchLogout}
+                      role="presentation"
+                    >
+                      Logout
+                    </span>
+                  </Popover>
+              )}
+                rootClose
               >
-                Login
-              </Nav.Item>
-              <Nav.Item
-                onClick={this.dispatchRegister}
-                className="nav-link auth-btn get-started"
-              >
-                Get Started
-              </Nav.Item>
-            </Nav>
-
-          )}
+                <img className="rounded-circle float-right space-icon" src="https://marriedbiography.com/wp-content/uploads/2018/01/Enrique-Iglesias.jpg" alt="user" />
+              </OverlayTrigger>
+            ) : (
+              <>
+                <Nav.Item
+                  onClick={this.dispatchLogin}
+                  className="nav-link auth-btn login"
+                >
+                  Login
+                </Nav.Item>
+                <Nav.Item
+                  onClick={this.dispatchRegister}
+                  className="nav-link auth-btn get-started"
+                >
+                  Get Started
+                </Nav.Item>
+              </>
```

Comment:
why do you have empty tags here `</>`?

Reply:
Replaced this with full `<React.Fragment>`. I think hound CI does not recognize the shorthand method.